data/Localization/german_(germany)/global.ini

```diff
@@ -43233,13 +43242,13 @@ crusader_hackprevent_desc_002=Das Datennetz bei ~mission(Location|Address) ist d
+crusader_platform_MultiEliminateAll_description_001=Nach ihrem letzten Versuch einer Belagerung haben die Nine Tails-Kräfte begonnen, gezielte Überfälle durchzuführen, um weiterhin die Operationen von Crusader zu stören. Der aktuelle Angriff hat mehrere Gebäude bei ~mission(MissionCluster|Address) ins Visier genommen, und es ist entscheidend, dass wir der Aggression dort ein Ende setzen.\n\nCrusader Security sucht einen Auftragnehmer, der bereit ist, alle Nine Tails-Operatoren in der Gegend zu beseitigen. Auftragnehmer sollten den Einsatz von Gewalt erwarten und werden dringend dazu ermutigt, Unterstützung in Betracht zu ziehen.\n\nAUFTRAG GENEHMIGT DURCH:\nVerbindungsoffizier Bautista\nID# 948J030K
 crusader_platform_MultiEliminateAll_title_001=Mehrere Angriffe auf Orison abwehren
-crusader_platform_MultiEliminateSpecific_desc=Crusader Security hat Nachrichtendienstinformationen erhalten, dass ein aktueller Überfall der Nine Tails auf ~mission(Location|Name) mehrere prominente Bandenvollstrecker umfasst. Die Beendigung dieses Überfalls und die Überführung der Verantwortlichen zur Rechenschaft ist eine oberste Priorität für Crusader Security. Deshalb suchen wir nach einem Auftragnehmer, der bereit ist, hinzugehen und diejenigen zu ergreifen, die ihn anführen.\n\nAn jedem Angriffsort müssen Sie die Kräfte der Nine Tails so lange dezimieren, bis Sie den Vollstrecker in Führung herausfordern können.\n\nIn Anbetracht der Brutalität, die von Nine Tails während ihrer jüngsten Belagerung eingesetzt wurde, empfehlen wir, dass Auftragnehmer darauf vorbereitet sein sollten, Gewalt anzuwenden, und wir ermutigen sie dringend, die Mitnahme von Unterstützung in Erwägung zu ziehen.\n\nAUFTRAG GENEHMIGT DURCH:\nVerbindungsoffizier Gibbs\nID# 91G66BW0
+crusader_platform_MultiEliminateSpecific_desc=Crusader Security hat Informationen erhalten, dass ein aktueller Überfall der Nine Tails, der bei ~mission(MissionCluster|Address) stattfindet, mehrere prominente Banden-Vollstrecker einschließt. Das Beenden dieses Überfalls und das Zur-Rechenschaft-Ziehen der Verantwortlichen ist für Crusader Security höchste Priorität. Deshalb suchen wir einen Auftragnehmer, der bereit ist, hinzugehen und die Anführer festzunehmen.\n\nAn jedem Angriffsort müssen Sie die Kräfte der Nine Tails dezimieren, bis Sie in der Lage sind, den zuständigen Vollstrecker herauszulocken.\n\nIn Anbetracht der Brutalität, die Nine Tails während ihrer jüngsten Belagerung eingesetzt hat, empfehlen wir den Auftragnehmern, sich auf den Einsatz von Gewalt vorzubereiten und dringend, Unterstützung mitzubringen.\n\nAUFTRAG GENEHMIGT DURCH:\nVerbindungsoffizier Gibbs\nID# 91G66BW0
```

Comment:
I think the mission variables are also wrong here, or why were they changed?
The same for line:
[43245](https://github.com/Dymerz/StarCitizen-Localization/pull/197/files#diff-05b4199920cf03182dc4c807896feb65de7908d4dc1e8fad4fcf64da3facf008R43245)
[43247](https://github.com/Dymerz/StarCitizen-Localization/pull/197/files#diff-05b4199920cf03182dc4c807896feb65de7908d4dc1e8fad4fcf64da3facf008R43247)
[43249](https://github.com/Dymerz/StarCitizen-Localization/pull/197/files#diff-05b4199920cf03182dc4c807896feb65de7908d4dc1e8fad4fcf64da3facf008R43249)

Reply:
They got changed in the English global.ini